types/ref-struct/index.d.ts

```diff
@@ -50,9 +50,9 @@ interface StructType extends ref.Type {
 
 /** The struct type meta-constructor. */
 declare var StructType: {
-    new (fields?: {}): StructType;
+    new (fields?: {}, opt?: {}): StructType;
```

Comment:
`{}` accepts all types, so it's as good as `any`. Did you mean `object` -- which allows all object types but rejects primitives (e.g. `string`)? If so , please do that -- or replace with `any`. Also please add a test case :smile:

Reply:
Opt is used to accept second parameter, right now only used to support C like packed structure,
please refer the source https://github.com/TooTallNate/ref-struct/blob/master/lib/struct.js#L125
A sample usage would be like:
```
const StructHardwareFeature = StructType({
  cmd: ref.types.uint8,
  bitmap: ref.types.uint8,
}, {packed: true});
```
The total size will be 2bytes

Reply:
Sorry. I was confusing. 
1. `{}` is not a good type to use, because any other type can be assigned to `{}`. For example, you can pass in `string` or `number` as the `opt` parameter. Right now, the intention is unclear.
  - If you want to accept any type, use `any`
  - If you want to only accept objects, use `object`.
2. Please add a test case.

Reply:
I changed {} to object as suggested, however for this change I need to add //TypeScript Version: 2.2 in the header, then the CI error comes:

ffi depends on ref-struct but has a lower required TypeScript version

Revert back to {} for now.

Reply:
I prefer to use `object` and add a min ts version to ffi to TS 2.2